definitions/pipelines/rnaseq_star_fusion.cwl

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "STAR-RNA-Seq alignment and transcript/gene abundance workflow"
+requirements:
+    - class: MultipleInputFeatureRequirement
+    - class: SubworkflowFeatureRequirement
+    - class: ScatterFeatureRequirement
+    - class: InlineJavascriptRequirement
+inputs:
+    instrument_data_bams:
+        type: File[]
+    outSAMattrRGline:
```

Comment:
I'm going to suggest using the `doc:` attribute to document some of these more confusing param names, specifically outSAMattrRGline and stargenomeDir.  The others are more straightforward and copied from the other rnaseq workflow (though we should document them all, eventually!)

Reply:
Something I've been wondering when considering these PRs: should we enforce our `snake_case` naming convention for inputs?

Reply:
+1  for that idea.  Let's keep things as consistent as possible!

Reply:
applied `snake_case` where ever necessary